neo/rawio/spikeglxrawio.py

```diff
@@ -331,10 +331,16 @@ def parse_spikeglx_fname(fname):
             stream_kind = None
         else:
             # the naming do not correspond lets try something more easy
+            # example: sglx_xxx.imec0.ap
             r = re.findall(r'(\S*)\.(\S*).(ap|lf)', fname)
             if len(r) == 1:
                 run_name, device, stream_kind = r[0]
                 gate_num, trigger_num = None, None
+            else:
+                # easy case for nidaq, example: sglx_xxx.nidq
+                r = re.findall(r'(\S*)\.(\S*)', fname)
+                run_name, device = r[0]
+                gate_num, trigger_num, stream_kind = None, None, None
```

Comment:
@guptadivyansh in this case shouldn't `stream_kind` be set to `nidq`? I guess this is the only case when we expect to get to this condition, right?
```suggestion
                stream_kind = "nidq"
                gate_num, trigger_num = None, None
```

Reply:
`stream_kind` was set to None for the normal NI case above (line 331), so I didn't change it. `device` will be set to `nidq` by the regex parser.

Reply:
@samuelgarcia what do you think?

Reply:
Actually, this is perfectly in line with the "default" behavior as you suggested. Resolving